src/python_testing/TC_EEVSE_Utils.py

```diff
@@ -0,0 +1,209 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+
+import logging
+import queue
+
+import chip.clusters as Clusters
+from chip.clusters import ClusterObjects as ClusterObjects
+from chip.clusters.Attribute import EventReadResult, SubscriptionTransaction
+from chip.interaction_model import InteractionModelError, Status
+from mobly import asserts
+
+logger = logging.getLogger(__name__)
+
+
+class EventChangeCallback:
```

Comment:
This looks generally quite helpful. Don't block this PR on it, but would you mind moving this into the base helper file so other folks could use it? It doesn't appear to be specific to this cluster

Reply:
I've add a follow Issue https://github.com/project-chip/connectedhomeip/issues/31460